src/integration/java/io/pinecone/integration/controlPlane/serverless/CreateDescribeListAndDeleteIndexTest.java

```diff
@@ -46,25 +43,6 @@ public void describeAndListIndex() {
         assertTrue(indexList.getIndexes().stream().anyMatch(index -> indexName.equals(index.getName())));
     }
 
-    @Test
```

Comment:
Just confirming, most of these test removals are covered by unit tests now, right?

Reply:
Correct!